news/2024/2024-06-03-the-followpoint-eriha-the-drum-decimator-of-osu-taiko.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "The Followpoint: Eriha, the Drum Decimator of osu!taiko"
```

Comment:
alternative title since "drum decimator" is an actual user title (and we might want to avoid the overlap)

```suggestion
title: "The Followpoint: Eriha, Top Drum Basher"
```

very open to suggestions